### DISCUSSION TITLE
Do you plan to highlight the line that caused the error or display the line number in the console?

### DISCUSSION THREAD

**Discussion by TakamiChie** [chinchang/web-maker#569] [Q&A] [answered]
I use WebMaker in my junior high school programming elective class.
In this class, students cannot create their own web service accounts or link their web accounts with external services, so this WebMaker mechanism is very useful because it can work offline and save data in local storage.

There are some things I would like to confirm before using WebMaker in this class again this year.
One of the bottlenecks in using WebMaker in the classroom is that when an error occurs, the line number of the part that caused the error may not be displayed, so it may take a lot of time for the teacher to find out the cause when the student is inquired.
If there is a mechanism that makes it easy to investigate the cause, such as highlighting error lines, we believe that it will be easier for teachers to respond.
Are there any plans to include features such as highlighting error lines or always displaying the line number of the line that caused the error in the console?

**Comment by chinchang** [comment, marked as answer]
Hi @TakamiChie 
Do you mean to say that error isn't highlighted everytime or not highlighted at all?
Confirming because we do highlight the line where error is present:

![CleanShot 2024-07-15 at 17 27 07](https://github.com/user-attachments/assets/95a1dc07-d19e-47e7-a85e-3eafdab98fd1)

**Reply by TakamiChie** [reply]
Thanks for the reply.
In some cases, the red mark was not displayed even if the semicolon was erased, so the red mark was not recognized as a highlight indicating an error.
In fact, when I checked again, the semicolon where the red mark was not displayed worked even if it was removed due to JavaScript syntax.
